site/content/installation/installing-nic/create-license-secret.md

```diff
@@ -0,0 +1,149 @@
+---
+docs: DOCS-602
+title: Create a License Secret 
+toc: true
+weight: 100
+---
+
+This document explains how to create and use a license secret for NGINX Ingress Controller. 
+
+# Overview
+
+NGINX Plus Ingress Controller requires a valid JSON Web Token (JWT) to download the container image from the F5 registry. From version 4.0.0, this JWT token is also required to run NGINX Plus.
+
+This requirement is part of F5’s broader licensing program and aligns with industry best practices. The JWT will streamline subscription renewals and usage reporting, helping you manage your NGINX Plus subscription more efficiently. The [telemetry](#telemetry) data we collect helps us improve our products and services to better meet your needs.
+
+The JWT is required for validating your subscription and reporting telemetry data. For environments connected to the internet, telemetry is automatically sent to F5’s licensing endpoint.  In offline environments, telemetry is routed through [NGINX Instance Manager](https://docs.nginx.com/nginx-instance-manager/). By default usage is reported every hour and also whenever NGINX is reloaded.
+
+{{< note >}}See [here](https://docs.nginx.com/solutions/about-subscription-licenses/#for-internet-connected-environments) for a list of IPs associated with F5's licensing endpoint (`product.connect.nginx.com`).{{</ note >}}
```

Comment:
```suggestion
{{< note >}} Read the [subscription licenses topic](https://docs.nginx.com/solutions/about-subscription-licenses/#for-internet-connected-environments) for a list of IPs associated with F5's licensing endpoint (`product.connect.nginx.com`). {{</ note >}}
```

We try to avoid having links that simply say "here" since there is an extra cognitive effort around contextualizing what it could be referring to: best to be explicit.